Add GitHub:Enterprise support to GitHub/notifications.30s.py

flesch opened the pull request:
I added support for fetching **GitHub:Enterprise** notifications. For those of us that use both **GitHub.com** and an enterprise instance, notifications from both places will show up (rather than running a second instance of this plugin with GH:E modifications).

![preview](https://cloud.githubusercontent.com/assets/13259/12300782/9f1f5ba8-b9e2-11e5-8e67-e59966aace9a.png)

If a user doesn't define a GH:E token and API, that section won't show up.

manojlds commented on Dev/GitHub/notifications.30s.py:
```diff
@@ -1,30 +1,71 @@
-# Fetch Github notifications in your menu bar!
+
+# <bitbar.title>GitHub Notifications</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
```
author metadata?

flesch replied:
What's the recommendation for a plugin with multiple authors (especially one with a change so drastic since @keithamus first wrote it)? I didn't want to assume Keith still wants to be on the hook to support a plugin that now looks much different.

matryer replied:
Only one author is currently supported on https://getbitbar.com/ - should we add support for comma separated authors?

keithamus replied:
@flesch I'm fine if you want to change the author to just you.

matryer replied:
Just have one in the `github` tag, but include both names (comma separated) in `author` as it's display only. That work?

flesch commented:
I updated the author metadata and squashed everything. I fixed an issue where it would incorrectly say "1 notifications" (like above) - is it worth updating the screenshot?

keithamus commented:
:+1: all good :smile:

flesch commented:
Thanks!
